bot/pagination.py

```diff
@@ -10,9 +10,9 @@
 LEFT_EMOJI = "\u2B05"    # [:arrow_left:]
 RIGHT_EMOJI = "\u27A1"   # [:arrow_right:]
 LAST_EMOJI = "\u23ED"    # [:track_next:]
-DELETE_EMOJI = "\u274c"  # [:x:]
+DELETE_EMOJI = "<:trashcan:637136429717389331>"  # [:trashcan:]
```

Comment:
The PR is good to be merged, but before that, let's move this to `constants.py` so we can easily change it later and will make sure that the trashcan used elsewhere will be the same as this one.

Reply:
Yeah, as I said, I will add it to the constants module.
Will it be a good idea to try to clear things out and try to solve #348?